### ISSUE TITLE
Write test cases angular/greek-mythology

### ISSUE THREAD

**Issue by vardhanapoorv** [vardhanapoorv/Open-Source-Contribution-Made-Easy#21]
Use testing framework of your choice.

**Comment by izulien** [comment]
What are you wanting to test for?  Do you have requirements for the project?  I would really like to practice writing up test cases but I'm not particularly familiar with writing these and may need some suggestions.

I would expect a test that verifies routes but there aren't any routes or real functions.  I could write a quick test that shows the god model being defined I suppose?

**Comment by vardhanapoorv** [comment]
@izulien yes the app currently pretty static and small. But later on it will integrated with backend data and we can add a card through UI. So keeping that in mind you can choose what you want to use for testing. Yes you can start with god model test. Feel free to ask me if you need more clarity.
